applications/nrf5340_audio/README.rst

```diff
@@ -87,13 +87,11 @@ Connected Isochronous Stream (CIS)
   In this configuration, you can use the nRF5340 Audio development kit in the role of the gateway, the left headset, or the right headset.
 
   In the current version of the nRF5340 Audio application, the CIS mode offers both unidirectional and bidirectional communication.
-  You can configure the CIS to bidirectional communication, in which it will support a walkie-talkie demonstration.
   In the bidirectional communication, the headset device will send audio from the on-board PDM microphone.
-  In the walkie-talkie demonstration, the gateway device will send audio from the on-board PDM microphone instead of using the line-in.
-  See `Enabling the walkie-talkie demo`_ for more information.
+  See `Selecting the CIS bidirectional communication`_ for more information.
 
-  .. note::
-     Only one headset device can be connected when testing the bidirectional mode or the walkie-talkie demo.
+  It is possible to enable a walkie-talkie demonstration. In this demonstration the gateway device will send audio from the on-board PDM microphone instead of using USB or the line-in.
```

Comment:
```suggestion
  You can also enable a walkie-talkie demonstration.
  In this demonstration, the gateway device will send audio from the on-board PDM microphone instead of using USB or the line-in.
```

Reply:
Changed